### PR TITLE
Removes edit() from RecipeActions

### PR DESCRIPTION
**Pull request by hermzz**

### CODE REVIEW

File: frontend/modules/recipe_form/components/RecipeForm.js [hunk 1]
```diff
@@ -65,12 +65,7 @@ class RecipeForm extends React.Component {
   }
 
   CreateRecipe(e) {
-    e.preventDefault();
```
Comment (RyanNoelk) [inline frontend/modules/recipe_form/components/RecipeForm.js:68]: We still need this to prevent the default behavior of the submit button.

**Review by RyanNoelk — CHANGES_REQUESTED** [review]
Small nit, Otherwise LGTM.

File: frontend/modules/recipe_form/components/RecipeForm.js (hunk 1 above)
Comment (hermzz) [reply frontend/modules/recipe_form/components/RecipeForm.js:68]: Good catch, fixed!

**Review by RyanNoelk — APPROVED** [review]
LGTM

**Pull request merged**